src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -115,6 +119,14 @@ public boolean belowMinimumAngle() {
         return getCurrentAngle() <= minAngleProp.get();
     }
 
+    public boolean LeftLimitStop(){
+        return motor.isRevLimitSwitchClosed();
+    }
+
+    public boolean rightLimitStop(){
```

Comment:
see above

Reply:
would be more clear if you made this `isAtLeftLimitStop` as John suggested earlier